opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -97,13 +97,24 @@ message ScopeMetrics {
 // - DataPoint contains timestamps, attributes, and one of the possible value type
 //   fields.
 //
-//     Metric
-//  +------------+
-//  |name        |
-//  |description |
-//  |unit        |     +------------------------------------+
-//  |data        |---> |Gauge, Sum, Histogram, Summary, ... |
-//  +------------+     +------------------------------------+
+// MetricsData
```

Comment:
Why provide a definition of the `MetricsData` type for the `Metric`? This seems more appropriate in the `MetricsData` type documentation.

Reply:
@jsuereth, @tigrannajaryan what do you think? Should I move this up?

Reply:
Works either for me, as long as we have this diagram somewhere.

Reply:
@tigrannajaryan, I just pushed a commit. Can you please take a look?